packages/overlays/package.json

```diff
@@ -32,7 +32,8 @@
     "*.js"
   ],
   "dependencies": {
-    "@lion/core": "^0.1.9"
+    "@lion/core": "^0.1.5",
+    "popper.js": "^1.15.0"
```

Comment:
After seeing the presentation you sent about security (https://www.youtube.com/watch?v=2cyib2MgvdM), maybe it's better to have only fixed versions for external dependencies we don't control?

Reply:
Doesn't outweigh importance of  preventing potential versio  conflicts with other deps loaded by App developer